README.md

```diff
@@ -1,2 +1,45 @@
-# toDo
-A web based application to organize and save your to do list.
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do
+
+> A simple minimalist browser based application to create save and manage your daily task.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Html/Css
+- Javascript
+- Webpack
+
+## Live Demo
+
+[Live Demo Link](https://amazing-shaw-03b76e.netlify.app)
+
```

Comment:
Good job setting up this README file.
If you check the [Microverse template](https://github.com/microverseinc/readme-template) you will see there is a section where you're supposed to add `Setup`.

Example: 
```
- git clone `your repo`
- cd into the folder
- git checkout `branch-name`
- run `npm i`
- run `npm start`
```

Reply:
- This section is added to the readme. Nice work. :clap: :clap: :clap: